### DISCUSSION TITLE
Support for foldable devices.

### DISCUSSION THREAD

**Discussion by pulsee** [FossifyOrg/General-Discussion#401] [Ideas]
Hello, 
I am doing research thesis on foldable devices for Android development. I would like to understand if this App supports foldable devices?
Thanks!

**Comment by Aga-C** [comment]
What do you mean? What's special in foldable devices, that support for it should be different from support for all other device types? Fossify Gallery should work flawlessly on different kinds of Android devices.

**Reply by pulsee** [reply]
@Aga-C I tested App on foldable. When you play video from gallery in fold/unfold state and change the state to unfold/fold it restarts video from beginning. Doesn't maintain state. UX must adapt based on posture of folds. You can check how google maps is redesigned based on foldable.

**Reply by pulsee** [reply]
Similarly if you open menu/dialog it doesn't preserve state

**Reply by Aga-C** [reply]
We rely on community contributions, especially for less popular devices, so if you see any bugs, please raise a [bug report](https://github.com/FossifyOrg/Gallery/issues/new?assignees=&labels=bug%2Cneeds+triage&projects=&template=bug_report.yml) (separate for each bug). If you encounter a bug in multiple Fossify apps, please raise a [bug report in General Discussions repository](https://github.com/FossifyOrg/General-Discussion/issues/new?assignees=&labels=bug%2Cneeds+triage&projects=&template=bug_report.yml).

**Reply by pulsee** [reply]
I asked because I wanted to contribute :)

**Comment by naveensingh** [comment]
There's now a general request related to this for all Fossify apps: https://github.com/FossifyOrg/General-Discussion/issues/363
